{"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"noCredentials","kind":"identifier"},{"kind":"text","text":": "},{"text":"CredentialsManagerError","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","kind":"typeIdentifier","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV"}],"languages":["swift"]}]}],"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/auth0\/credentialsmanagererror\/nocredentials"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"schemaVersion":{"patch":0,"minor":3,"major":0},"abstract":[{"type":"text","text":"No credentials were found in the store."},{"type":"text","text":" "},{"type":"text","text":"This error does not include a "},{"type":"reference","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","isActive":true},{"type":"text","text":"."}],"kind":"symbol","metadata":{"symbolKind":"property","modules":[{"name":"Auth0"}],"fragments":[{"kind":"keyword","text":"static"},{"text":" ","kind":"text"},{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"preciseIdentifier":"s:5Auth023CredentialsManagerErrorV","text":"CredentialsManagerError","kind":"typeIdentifier"}],"externalID":"s:5Auth023CredentialsManagerErrorV02noB0ACvpZ","role":"symbol","roleHeading":"Type Property","title":"noCredentials"},"references":{"doc://com.auth0.Auth0/documentation/Auth0/Auth0Error/cause-9wuyi":{"role":"symbol","abstract":[{"text":"The underlying ","type":"text"},{"code":"Error","type":"codeVoice"},{"text":" value, if any. Defaults to ","type":"text"},{"code":"nil","type":"codeVoice"},{"type":"text","text":"."}],"title":"cause","kind":"symbol","url":"\/documentation\/auth0\/auth0error\/cause-9wuyi","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cause","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"kind":"text","text":"?"}],"type":"topic"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError/noCredentials":{"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials","abstract":[{"type":"text","text":"No credentials were found in the store."},{"text":" ","type":"text"},{"type":"text","text":"This error does not include a "},{"isActive":true,"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","type":"reference"},{"type":"text","text":"."}],"type":"topic","kind":"symbol","url":"\/documentation\/auth0\/credentialsmanagererror\/nocredentials","title":"noCredentials","role":"symbol","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"noCredentials","kind":"identifier"},{"kind":"text","text":": "},{"text":"CredentialsManagerError","kind":"typeIdentifier","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV"}]},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError":{"fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"CredentialsManagerError","kind":"identifier"}],"abstract":[{"text":"Represents an error during a Credentials Manager operation.","type":"text"}],"url":"\/documentation\/auth0\/credentialsmanagererror","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","role":"symbol","kind":"symbol","type":"topic","title":"CredentialsManagerError","navigatorTitle":[{"kind":"identifier","text":"CredentialsManagerError"}]},"doc://com.auth0.Auth0/documentation/Auth0":{"kind":"symbol","url":"\/documentation\/auth0","role":"collection","title":"Auth0","type":"topic","abstract":[{"text":"SDK for Apple platforms.","type":"text"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0"}}}